collector.go

```diff
@@ -315,14 +326,40 @@ func pduToSamples(indexOids []int, pdu *gosnmp.SnmpPDU, metric *config.Metric, o
 		// It's some form of string.
 		t = prometheus.GaugeValue
 		value = 1.0
+		metricType := metric.Type
+
+		// For InetAddress, lookup the type and substitute it in.
+		if metric.Type == "InetAddress" {
```

Comment:
I think this should be `metricType`, as we just set it above.

Reply:
Done